src/main/java/uk/gov/justice/digital/hmpps/csr/api/config/RegionAwareRoutingSource.kt

```diff
@@ -1,12 +1,9 @@
 package uk.gov.justice.digital.hmpps.csr.api.config
 
 import org.springframework.jdbc.datasource.lookup.AbstractRoutingDataSource
-import uk.gov.justice.digital.hmpps.csr.api.utils.RegionContext.getRegion
+import uk.gov.justice.digital.hmpps.csr.api.utils.RegionContext
 
 class RegionAwareRoutingSource : AbstractRoutingDataSource() {
-  override fun determineCurrentLookupKey(): Any {
-    // Default to 1, only to avoid NPE
-    // not because region 1 is special.
-    return getRegion() ?: "1"
-  }
+
+  override fun determineCurrentLookupKey(): Any = RegionContext.getRegion().toString()
```

Comment:
No longer assume region 1
Now the legacy endpoints fail with a 500 error if the region is missing which is a bit rough and ready but they will be removed once the new regime is live